CHANGELOG.txt

```diff
@@ -2,6 +2,17 @@
 TextWiser CHANGELOG
 =====================
 
+-------------------------------------------------------------------------------
+Mar 03, 2022 1.4.0
+-------------------------------------------------------------------------------
+major:
+- Update UMAP to return deterministic output in latest version
+- Update SVD to return deterministic output
+
+minor:
+- Disable non-relevant transformers warning
+- Add full requirement install option
```

Comment:
what does "full" requirement mean? add some mention here

Reply:
oh I see. it points to full_req.txt.. some may be explicitly say full_requirements.txt so that people know where it refers to